themes/tdr/login/login.ftl

```diff
@@ -0,0 +1,39 @@
+<#import "template.ftl" as layout>
+<@layout.registrationLayout; section>
+    <#if section = "header">
+        ${msg("doLogIn")}
+    <#elseif section = "form">
+        <#if realm.password>
+            <form id="kc-form-login" onsubmit="login.disabled = true; return true;" action="${url.loginAction}" method="post">
+                <div class="govuk-form-group">
+                    <h1 class="govuk-label-wrapper">
+                        <label for="username" class="govuk-label govuk-label--l">
+                            <#if !realm.loginWithEmailAllowed>${msg("username")}
+                            <#elseif !realm.registrationEmailAsUsername>${msg("usernameOrEmail")}
+                            <#else>${msg("email")}
+                            </#if>
+                        </label>
+                    </h1>
+                    <#if usernameEditDisabled??>
```

Comment:
Ditto for this `if` statement. I don't think we want users to change their usernames.